renovate.json5

```diff
@@ -66,4 +66,14 @@
       groupName: 'prettier',
     },
   ],
+
+  // A list of dependencies to be ignored by Renovate - "exact match" only
+  ignoreDeps: [
```

Comment:
One implication of this is that we'll never get renovate PRs. If this is going to be temporary this is fine. Another option is to just set a package rule for these deps to have automerge turned off. That way we'd get PRs but they won't be automerged by renovate, but someone would need to remember to close those PRs to unblock others and not merge them, which doesn't seem ideal either.

Reply:
Hopefully this is temporary just until ACT-962 completed 🤞.